### PR TITLE
Base64#urlsafe_encode now works as in ruby, fixed #3394

### LINKED ISSUE #3394: Base64#urlsafe_encode why padding false by default?
is there any reason for padding be false?

because ruby cannt decode this:
```crystal
crystal eval 'require "base64"; puts Base64.urlsafe_encode "bl"'
Ymw
```

```ruby
irb: Base64.urlsafe_decode64 "Ymw"
ArgumentError: invalid base64
```

if i add padding, it works

### PR DESCRIPTION
**Pull request by kostya**

### CODE REVIEW

File: src/base64.cr [hunk 1]
```diff
@@ -149,11 +149,8 @@ module Base64
   # Alphabet" in RFC 4648.
   #
   # The alphabet uses '-' instead of '+' and '_' instead of '/'.
-  #
-  # The `padding` parameter defaults to false. When true, enough `=` characters
-  # are added to make the output divisible by 3.
   def urlsafe_encode(data, io : IO)
```
Comment (asterite) [inline src/base64.cr]: Should we have `padding = true` here too?

**Comment by asterite** [comment]
Looks good, thanks! I added a small question.

File: src/base64.cr (hunk 1 above)
Comment (kostya) [inline src/base64.cr]: i want to remove padding option, and i thought this is not needed, but used in secure_random
https://github.com/crystal-lang/crystal/blob/master/src/secure_random.cr#L46

in io method, i not think it needed

Comment (asterite) [inline src/base64.cr]: OK. We can always add it later if needed :-)

**Pull request merged**